spec/services/actual/export_spec.rb

```diff
@@ -2,77 +2,64 @@
   let(:partner_organisation) { create(:partner_organisation) }
```

Comment:
The commit that changed this file can be dropped if we want. I was barking up the wrong tree when I changed this, I thought the fields needed removing from the "actuals" export as well, but it seems that that includes all activities types. However, I think the added test coverage might help someone in future.